src/zkevm_specs/evm/execution/copy_code_to_memory.py

```diff
@@ -0,0 +1,60 @@
+import itertools
+from typing import Iterator
+
+from ...util import FQ, N_BYTES_MEMORY_SIZE, RLC
+from ..execution_state import ExecutionState
+from ..instruction import Instruction, Transition
+from ..step import CopyCodeToMemoryAuxData
+from ..table import RW
+from ..util import BufferReaderGadget
+
+
+MAX_COPY_BYTES = 54
```

Comment:
Please, if this constant is not already present in the repo, document it and import it from `util`